### ISSUE TITLE
Puyo Puyo Fever 2 broken image

### ISSUE THREAD

**Issue by IrfanH495** [hrydgard/ppsspp#18941]
### Game or games this happens in

ULJM-05058

### What area of the game

at school and choose waku waku and hara hara.

### What happens

OpenGL
![Screenshot_20240321-051912_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/d24577d8-982d-46f8-9e38-ec45fe1eacd4)

Vulkan
![Screenshot_20240321-051856_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/27057a55-fc26-440c-8aa1-52758fb19ceb)

software rendering
![Screenshot_20240321-051925_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/179cae3e-86ca-44a4-87f0-315020e5b2f3)

### What should happen

sorry I checked this I can't create an issue
![20240321_054042](https://github.com/hrydgard/ppsspp/assets/78854859/c98d2dd7-f050-414d-9482-a0f8822b6a0d)

### GE frame capture

[ULJM05058_0001.zip](https://github.com/hrydgard/ppsspp/files/14673191/ULJM05058_0001.zip)

### Platform

Android

### Mobile device model or graphics card (GPU)

Samsung M20 Exynos 7904 Mali-G71 MP2

### PPSSPP version affected

v1.17.1-35-

### Last working version

I do not know

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the glitch (especially speed hacks or enhancements/replacements.)
- [X] Include logs or screenshots of issue.

**Comment by IrfanH495** [comment]
Coincidentally, I still have PPSSPP v1.1.1 and I tried it and hope the issue doesn't exist
like opengl in the latest ppsspp


https://github.com/hrydgard/ppsspp/assets/78854859/14d582bf-25ab-44ec-8266-b64a8d30c8f0

**Comment by anr2me** [comment]
Probably similar issue to https://github.com/hrydgard/ppsspp/issues/18927
Might be GPU-specific issue?

You should provide a GE frame dump for testing purpose.

**Comment by hrydgard** [comment]
Interesting, yeah, probably Mali-specific somehow.

**Comment by IrfanH495** [comment]
yes only in Mali, I tried it on sharp sh04h didn't experience it
Vulkan
![20240322_022918](https://github.com/hrydgard/ppsspp/assets/78854859/c636b295-5aa2-4abc-bd12-1471403c30d4)


opengl
![20240322_022940](https://github.com/hrydgard/ppsspp/assets/78854859/1ff207c5-2496-40e2-a8fc-770c46b0fc5c)

**Comment by anr2me** [comment]
Btw have you tried using PixelDepthRounding compatibility?

**Comment by IrfanH495** [comment]
> Btw have you tried using PixelDepthRounding or VertexDepthRounding compatibility?

if only PixelDepthRounding the issue still exists so I added it to DisableAccurateDepth, the issue no longer exists.
but it only applies to vulkan, opengl still experiences it I don't know why.
